packaging/hudi-timeline-server-bundle/run_server.sh

```diff
@@ -19,13 +19,35 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 #Ensure we pick the right jar even for hive11 builds
-HOODIE_JAR=`ls -c $DIR/target/hudi-timeline-*.jar | grep -v test | head -1`
+HOODIE_JAR=`ls -c $DIR/target/hudi-timeline-server-bundle-*.jar | grep -v test | head -1`
```

Comment:
move this also to REPO/scripts?

Reply:
@vinothchandar I think there are other scripts like this (For e:g run_hive_sync). Let me create a jira and do all of this move separately. Other changes also involves updating demo documentation.

Reply:
Filed HUDI-274